docs/feature/subnet-scarcity/phase-1/1-subnetstate.md

```diff
@@ -0,0 +1,83 @@
+## DNC SubnetState API and Subnet Utilization cache [[Phase 1 Design](../proposal.md#1-1-subnet-utilization-is-cached-by-dnc)]
+
+### SubnetState API
+An API will be added to DNC which will provide the Reserved IP Count (the "Utilization") of a Subnet. The API will synchronously query the Subnet Utilization Cache and return the response from the Cache directly.
+
+```yaml
+paths:
+  /networks/{networkID}/subnets/{subnet}/utilization:
+    get:
+      summary: Returns the Subnet State
+      operationId: querySubnetState
+      description: |
+        Queries the State for the passed Subnet.                             
+      parameters:
+        - in: path
+          name: networkID
+          description: The Network ID
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: subnet
+          description: The Subnet Name
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: The matching SubnetState
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/SubnetState'
+        '400':
+          description: bad input parameter
```

Comment:
@rsagasthya can you fill in the missing response types and codes of this API as it has been written?